cmd/main.go

```diff
@@ -486,3 +488,12 @@ func setupMandatoryModuleDeletionReconciler(mgr ctrl.Manager,
 		os.Exit(bootstrapFailedExitCode)
 	}
 }
+
+func setupIstioReconciler(mgr ctrl.Manager, flagVar *flags.FlagVar, options ctrlruntime.Options, setupLog logr.Logger) {
+	options.MaxConcurrentReconciles = flagVar.MaxConcurrentKymaReconciles
```

Comment:
what about using the watcher reconciles and requeue intervals as well for this controller?